src/outputs/outputs.hpp

```diff
@@ -46,26 +46,19 @@ struct OutputParameters {
   int file_number_width;
   bool file_label_final;
   std::string file_id;
-  std::string variable;
```

Comment:
As far as I can tel this is completely unused now. It was used in the VTK output but I think it shouldn't be as that's the wrong scope with our data model.